_data/people/SebastianMacaluso.yml

```
@@ -64,8 +64,9 @@ presentations:
     focus-area: ia
   - title: "Looking into Jets with Machine Learning"
     date: 2020-01-17
-    url: https://indico.cern.ch/event/809820/overview
-    meeting: ML4Jets2020
-    meetingurl: https://indico.cern.ch/event/809820/overview
+    url: https://indico.cern.ch/event/809820/contributions/3632649/attachments/1971192/3279097/MacalusoML4Jets2020.pdf
```

```suggestion
    url: https://indico.cern.ch/event/809820/contributions/3632649
```

I don't think the remaining portion of the Indico link is stable.